src/main/java/com/autilus/hw1006/Node.java

```diff
@@ -0,0 +1,11 @@
+package com.autilus.hw1006;
+
+public class Node<T> {
```

Comment:
better make this class private static nested

Reply:
done